@radcortez I don't think `UnknownBuildConfigTest` should fail like that?
```
[INFO] Running io.quarkus.extest.UnknownBuildConfigTest
The output of the Quarkus build can be found at D:\a\quarkus\quarkus\integration-tests\test-extension\extension\deployment\target\quarkus-prod-mode-test4665286982218461875\build-result
2022-03-31 22:08:19,071 INFO  [org.jbo.threads] (main) JBoss Threads version 3.4.2.Final
2022-03-31 22:08:19,152 INFO  [io.qua.ext.dep.TestProcessor] (build-47) boot, launchMode=NORMAL
2022-03-31 22:08:19,250 WARN  [io.qua.dep.ste.BannerProcessor] (build-46) Unable to read banner file
2022-03-31 22:08:19,257 INFO  [io.qua.ext.dep.TestProcessor] (build-40) loadDSAPublicKey run
2022-03-31 22:08:19,678 INFO  [io.qua.ext.dep.TestProcessor] (build-21) Have XmlConfig, loading
2022-03-31 22:08:19,704 INFO  [io.qua.ext.dep.TestProcessor] (build-21) Loaded XmlConfig, creating service
2022-03-31 22:08:19,726 INFO  [io.qua.ext.dep.TestProcessor] (build-57) Registering service start
2022-03-31 22:08:19,903 WARN  [io.qua.config] (build-3) Unrecognized configuration key "quarkus.build.unknown.prop" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2022-03-31 22:08:19,903 WARN  [io.qua.config] (build-3) Unrecognized configuration key "quarkus.rename-old.only-in-new" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2022-03-31 22:08:19,903 WARN  [io.qua.config] (build-3) Unrecognized configuration key "quarkus.unknown.prop" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2022-03-31 22:08:19,903 WARN  [io.qua.config] (build-3) Unrecognized configuration key "quarkus.http.non.application.root.path" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2022-03-31 22:08:19,903 WARN  [io.qua.config] (build-3) Unrecognized configuration key "quarkus.rename-old.only-in-old" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2022-03-31 22:08:19,904 WARN  [io.qua.config] (build-3) Unrecognized configuration key "quarkus.http.ssl.port" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2022-03-31 22:08:19,904 WARN  [io.qua.config] (build-3) Unrecognized configuration key "quarkus.rename-old.prop" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2022-03-31 22:08:19,904 WARN  [io.qua.config] (build-3) Unrecognized configuration key "quarkus.rename-old.in-both" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2022-03-31 22:08:19,904 WARN  [io.qua.config] (build-3) Unrecognized configuration key "quarkus.rename-old.with-default" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2022-03-31 22:08:19,904 WARN  [io.qua.config] (build-3) Unrecognized configuration key "quarkus.arc.unremovable-types[0]" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2022-03-31 22:08:20,937 INFO  [DSAPublicKeyObjectSubstitution] (build-61) DSAPublicKeyObjectSubstitution.serialize
2022-03-31 22:08:21,217 INFO  [io.qua.dep.QuarkusAugmentor] (main) Quarkus augmentation completed in 2730ms
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 4.301 s <<< FAILURE! - in io.quarkus.extest.UnknownBuildConfigTest
[ERROR] io.quarkus.extest.UnknownBuildConfigTest.unknownBuildConfig  Time elapsed: 0.002 s  <<< ERROR!
java.lang.NullPointerException
	at java.base/java.util.Objects.requireNonNull(Objects.java:221)
	at java.base/java.util.Arrays$ArrayList.<init>(Arrays.java:4323)
	at java.base/java.util.Arrays.asList(Arrays.java:4310)
	at io.quarkus.extest.UnknownBuildConfigTest.lambda$unknownBuildConfig$1(UnknownBuildConfigTest.java:32)
```

--- a/integration-tests/test-extension/extension/deployment/disable-unbind-executions
+++ b/integration-tests/test-extension/extension/deployment/disable-unbind-executions
@@ -1,0 +1,1 @@
+This file disables the unbind-executions profile in the quarkus-integration-tests-parent.

--- a/integration-tests/test-extension/extension/disable-unbind-executions
+++ b/integration-tests/test-extension/extension/disable-unbind-executions
@@ -1,0 +1,1 @@
+This file disables the unbind-executions profile in the quarkus-integration-tests-parent.

--- a/integration-tests/test-extension/extension/runtime/disable-unbind-executions
+++ b/integration-tests/test-extension/extension/runtime/disable-unbind-executions
@@ -1,0 +1,1 @@
+This file disables the unbind-executions profile in the quarkus-integration-tests-parent.

